src/main/java/pl/isa/biblioteka/Book.java

```diff
@@ -10,7 +10,7 @@ public class Book {
     private String author;
     @JsonProperty("kind")
     private String category;
-    @JsonIgnore
+//    @JsonIgnore
```

Comment:
Zamiast komentować adnotację, na przyszłość moglibyście pomyśleć o posiadaniu 2 klas Book, 1 do odczytywania danych z API, 2 z polami, które będą wykorzystywane przez Was w programie. Ale to później też na warsztatach będzie dokładnie omówione w ramach DTO, itp.